### ISSUE TITLE
Lockups when changing folders and Samba share open in the other panel

### ISSUE THREAD

**Issue by michalkochanowicz** [doublecmd/doublecmd#1986]
### Version

Double Commander
Version: 1.1.20 gamma
Revision: 607
Commit: d7d098956
Build date: 2024/11/10
Lazarus: 3.7.0.0
FPC: 3.2.2
Platform: aarch64-Darwin-cocoa
OS version: Mac OS X 15.0.1

### Description

This happens only if there is a Samba share in any of the panels. This is normal share, mounted by Mac to /Volumes.

When I'm changing folders in the panel with local drive, every other change is very long (more than 10 seconds). Mouse pointer is changed to "beach ball" and Double Commander is not responding.

I can not replicate this with any other file manager (Finder etc.).

You can see this in this video: https://youtu.be/L5ubFvekWbE

What happens:
1. There is a Samba share in the right panel.
2. There is a local drive in the left panel.
3. I'm moving to parent folder in right panel with the keyboard. This is quick.
4. I'm trying to enter "jdk1.8.0_401.jdk" folder and it takes ~30 seconds.
5. In the background there is Samba log open. There is no activity most of the time.

When I'm moving mouse pointer in circles, I'm seeing a "beach ball" and Double Commander is not responding.

**Comment by chachizhu** [comment]
I have same issue.
But when I disable ```Auto Refresh``` feature, it works like charm.

**Comment by rich2014** [comment]
@alexx2000 

Alex, any ideas?

**Comment by alexx2000** [comment]
No, debugging needed.

**Comment by michalkochanowicz** [comment]
@alexx2000, can I help somehow with debugging?

**Comment by StephanRoemer** [comment]
Yep, having the same issue. As @michalkochanowicz already mentioned: is there any way we can help with the debugging process?
